cirq-ionq/cirq_ionq/ionq_native_gates_test.py

```diff
@@ -74,3 +74,74 @@ def test_ms_unitary(phases):
 
     mat = cirq.protocols.unitary(gate)
     numpy.testing.assert_array_almost_equal(mat.dot(mat.conj().T), numpy.identity(4))
+
+
+@pytest.mark.parametrize(
+    "gate,target,phases",
+    [
+        (ionq.GPIGate, numpy.identity(2), {"phi": p})
+        for p in [0, 0.1, 0.4, math.pi / 2, math.pi, 2 * math.pi]
+    ]
+    + [
+        (ionq.GPI2Gate, numpy.identity(2), {"phi": p})  # type: ignore
+        for p in [0, 0.1, 0.4, math.pi / 2, math.pi, 2 * math.pi]
+    ]
+    + [
+        (ionq.MSGate, numpy.identity(4), {"phi0": p0, "phi1": p1})  # type: ignore
+        for p0, p1 in [
+            (0, 1),
+            (0.1, 1),
+            (0.4, 1),
+            (math.pi / 2, 0),
+            (0, math.pi),
+            (0.1, 2 * math.pi),
+        ]
+    ],
+)
+def test_gate_inverse(gate, target, phases):
```

Comment:
Since all targets are identity, you should be able to compute the appropriate matrix inside the test.